config/lib/seed.js

```diff
@@ -1,381 +1,406 @@
 					if (users.length > 0) {
 						users[0].password = user.password;
 						users[0].save(users[0], function(err, user) {
 							if (err) {
 								console.error(err);
 								reject(new Error('Failed to update existing user ' + user.username));
-							}
-							else {
+							} else {
 								resolve(user);
 							}
 						});
 					}
 					// if we didn't find an existing user, create the new user
 					else {
-						user.save(function (err, newUser) {
+						user.save(function(err, newUser) {
 							if (err) {
-								console.log (err);
+								console.log(err);
 								reject(new Error('Failed to add local ' + user.username));
-							}
-							else {
+							} else {
 								resolve(newUser);
 							}
 						});
 					}
 				}
-			})
+			});
 		});
 	};
 }
 
-function reportSuccess (password) {
-	return function (user) {
-		return new Promise(function (resolve, reject) {
+function reportSuccess(password) {
+	return function(user) {
+		return new Promise(function(resolve, reject) {
 			if (seedOptions.logResults) {
-				console.log(chalk.yellow('Database Seeding:\tLocal user \'' + user.username + '\' has password set to \'' + password + '\''));
+				console.log(chalk.yellow("Database Seeding:\tLocal user '" + user.username + "' has password set to '" + password + "'"));
 			}
 			resolve();
 		});
 	};
 }
 
 // save the specified user with the password provided from the resolved promise
-function seedTheUser (user) {
-	return function (password) {
-		return new Promise(function (resolve, reject) {
-
+function seedTheUser(user) {
+	return function(password) {
+		return new Promise(function(resolve, reject) {
 			// set the new password
 			user.password = password;
 
 			Promise.resolve()
-			.then(saveUser(user))
-			.then(reportSuccess(password))
-			.then(function () {
-				resolve();
-			})
-			.catch(function (err) {
-				resolve(err);
-			});
+				.then(saveUser(user))
+				.then(reportSuccess(password))
+				.then(function() {
+					resolve();
+				})
+				.catch(function(err) {
+					resolve(err);
+				});
 		});
 	};
 }
 
-function clearTemplates () {
-	var T = mongoose.model ('MessageTemplate');
-	return new Promise (function (resolve, reject) {
-		T.remove ({}, function (err) {
+function clearTemplates() {
+	var T = mongoose.model('MessageTemplate');
+	return new Promise(function(resolve, reject) {
+		T.remove({}, function(err) {
 			if (err) {
 				console.error(err);
 				console.error('Error removing templates');
 				reject();
 			}
-			resolve ();
+			resolve();
 		});
 	});
 }
 
-function seedTestMessageTemplate () {
+function seedTestMessageTemplate() {
 	console.log(chalk.yellow('Database seeding:\tSeeding message templates.'));
-	var T = mongoose.model ('MessageTemplate');
-	var saveT = function (t) {
-		return new Promise (function (resolve, reject) {
-			t.save (function (err) {
+	var T = mongoose.model('MessageTemplate');
```

Comment:
I'd suggest replace `T` with `MessageTemplate` just as a convention since other Mongoose model declarations do that

Reply:
Good suggestion, but we won't do that refactoring right now.